cos/static/player_join.js

```diff
@@ -166,7 +166,7 @@ function wait_for_players_to_join() {
     get_players_in_game(function(data){
 
         var current_player_count = data.Players.length;
-
+        document.getElementById("gameName").innerHTML = data.Game.game_name;
```

Comment:
Since the game name doesn't change, it probably would be better to set this at the beginning of rendering the page in the views.py file. Otherwise this would be rendering the every time this is called, which is unnecessary load. See my other comment below.